server/middlewares/partyMiddleware.js

```diff
@@ -7,6 +7,15 @@ class PartyMiddleware {
     }
     return next();
   }
+
+  static getAParty(req, res, next) {
+    const { id } = req.params;
+    if (isNaN(parseInt(id, 10))) {
```

Comment:
@frankly034 Good work so far! please use the `Number.isNaN` as against the `isNaN` global method you used

Reply:
Thank you very much. Was thinking what that eslint error meant.

Reply:
Resolved some conflicts.